README.adoc

```diff
@@ -1368,13 +1368,17 @@ The `and` and `or` keywords are banned.
 The minimal added readability is just not worth the high probability of introducing subtle bugs.
 For boolean expressions, always use `&&` and `||` instead.
 For flow control, use `if` and `unless`; `&&` and `||` are also acceptable but less clear.
+Please note that the `and` and `or` operators have lower precedence than the `=` operator, whereas the `&&` and `||` operators have higher precedence than the `=` operator, based on order of operations.
```

Comment:
I'd probably mention here also that `&&` has higher precedence than `||`, but `and` and `or` have the same one, which IMO is another source of confusion and potential bugs.

Reply:
@bbatsov 
Good point. I've updated the pull request.